### PR TITLE
feat(lm-logs): support bearer token for authentication

### PR DESCRIPTION
**Pull request by siddharthck**
Bearer token support for lm-logs chart.
Support bearer token for authentication with LogicMonitor with lm-logs. 
Bump up lm-logs chart version, app version.

### CODE REVIEW

**Review by sagarmamdapure — APPROVED** [review]

**Comment by sagarmamdapure** [comment]
@siddharthck Kindly wait for others approval before merging.

**Comment by siddharthck** [comment]
@vkumbhar94 @shubhashri12 can you have a look at this PR ?

**Review by vkumbhar94 — APPROVED** [review]

**Review by shubhashri12 — APPROVED** [review]

**Pull request merged**
